runtime/scripts.vim

```diff
@@ -306,7 +306,9 @@ else
     set ft=virata
 
     " Strace
-  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve(' || s:line1 =~# '^__libc_start_main'
+  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve('
+    \ || s:line1 =~# '^__libc_start_main'
+    \ || s:line1 =~# 'execve(.*)\s\+= \d\+'
```

Comment:
> The first regex: ^\(\[pid \d\+\] \)\=[0-9:.]* *execve( should already match the part you added

Well, it doesn't. I suspect it is probably because the `\=` part in the first part. To be honest, I don't really understand that part of the regex -- the way I read the current regex is "match an optional pid part (which may contain the word pid, space, one or more numbers), followed by a space, then, match a *non-optional* equals symbol ?, then the timestamp (which may be zero or more number or colon or period), followed by the execve part..." .

In any case, this is fairly trivial to test using the examples I provided in the commit message. Without the patch, the ft is not set and with it, it does get set correctly.